src/api/integrations/chatbot/typebot/services/typebot.service.ts

```
@@ -1,96 +1,215 @@
+        } else {
+          await this.prismaRepository.integrationSession.deleteMany({
+            where: { botId: bot.id, remoteJid: remoteJid },
+          });
+        }
+        return;
+      }
+
+      // Continue an existing chat
+      const version = this.configService?.get<Typebot>('TYPEBOT').API_VERSION;
       let url: string;
       let reqData: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)